### DISCUSSION TITLE
Notification widget font size

### DISCUSSION THREAD

**Discussion by knightsg** [breezy-weather/breezy-weather#434] [Q&A] [answered]
Is it possible to enlarge the font size used to display the temperature in the notification widget? It's quite small and I don't see any option to enlarge it.

Screenshot for reference: [https://imgbox.com/Z82tqsgo](https://imgbox.com/Z82tqsgo)

**Comment by papjul** [comment, marked as answer]
It’s not a customizable text, it’s an icon image (square shape). Because "15°" has a greater width than height, it has padding on top and bottom (otherwise it would be cropped). I will try to improve it a bit for single digit degrees on next version, but it won’t change the fact that Android expects a square icon (we can’t use pure text or a rectangle icon image).
